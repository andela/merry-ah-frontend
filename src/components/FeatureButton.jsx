import React from 'react';
import PropTypes from 'prop-types';

const FeatureButton = ({ text, myclass }) => (<button className={myclass}>
  {text}
</button>);

FeatureButton.propTypes = {
  text: PropTypes.string.isRequired,
  myclass: PropTypes.string.isRequired,
};

export default FeatureButton;
