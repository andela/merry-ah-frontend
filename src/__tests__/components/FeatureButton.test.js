/* eslint-disable import/no-unresolved */
import React from 'react';
import { shallow } from 'enzyme';
import FeatureButton from '../../components/FeatureButton';

let wrapper;
beforeEach(() => {
  wrapper = shallow(<FeatureButton text ='active' myclass='cardactive' />);
});
describe('FeatureButton Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
