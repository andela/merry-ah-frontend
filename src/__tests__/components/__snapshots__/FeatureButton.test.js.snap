// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeatureButton Component should match snapshot 1`] = `
<button
  className="cardactive"
>
  active
</button>
`;
